website/_posts/2023-03-31-what-is-testnet.md

```diff
@@ -0,0 +1,52 @@
+
+## What is Testnet?
+A Testnet is a simulation of a Blockchain, in which developers can test all the functionalities of their Blockchain network without risking their assets and convenience of end-users. Using Testnet, any possible error, bug or malfunctions can be fixed before the Mainnet launch.
+Testnet also can be an experimental environment for new features, updates and upgrades. So several testnet networks of the same version or newer version of the Mainnet can be implemented on the roadmap.
+
+
+## What are use cases of testnet?
+**A secure environment to test development ideas and updates.** So 	many features and minor details in Pactus project have been 	successfully developed so far using the Testnet; such as:
+
+- extra details and information on Pactus-GUI;
+ 	
+- well-functioning 	Pactus Blockchain explorer to access transactions and block details;
+ 	
+- Upgrades 	that led to decreasing the size of the Pactus network
+
+**Providing 	a safe testing ground for resolve scalability and security problems, 	as well as decentralization issues.** Pactus 	network is also a secure and purely decentralized network. However 	in the last updates during the Testnet, 	developers are now 	able to decrease 	the database size by approximately 10% without compromising 	performance. They did it by utilizing block height instead of hash 	to store blocks within the database. (you can find out more here).
+ 	
+**Testing 	new versions, patches and features on testnet.** 	it is vital for enhancing the security, load testing, Blockchain 	migration, integration tests, and etc. This is why you can now use 	the latest version of Pactus-GUI that works like clockwork and is 	extremely secure.
+ 	
+**Testing 	the functions of cryptocurrency without disrupting the mainnet.** In 	the testnet of Pactus Blockchain, you as a user or validator can use 	all the features to send, receive, bond or unbond your assets in the 	pactus-GUI.
+ 	
+**Development 	and simulation of smart contract features and DApps.** 	Note that many Blockchain projects like Pactus, can be used as 	smart-contract platforms. So the testnet can be designed in 	different stages to test and improve different applications of the 	network.
+
+Please note that after a certain period of time, when developers get satisfied with the functionality and debugging process, they launch Mainnet. Mainnet has its own independent genesis blocks; so the assets and addresses that users use in Testnet are not valid inside the new mainnet Blockchain anymore. In fact the whole Blockchain network starts working from the scratch and users and validators start to work with real tokens and transactions instead of the simulated ones. But since testnet has an important effect on the health and improvement of the network, in Pactus Blockchain, we have several advantages for the people who join the testnet.
```

Comment:
```suggestion
Please note that after a certain period of time, when developers get satisfied with the functionality and debugging process, they launch Mainnet. Mainnet has its own independent genesis blocks; so the assets and addresses that users use in Testnet are not valid inside the new Mainnet Blockchain anymore. In fact the whole Blockchain network starts working from the scratch and users and validators start to work with real tokens and transactions instead of the simulated ones. But since Testnet has an important effect on the health and improvement of the network, in Pactus Blockchain, we have several advantages for the people who join the Testnet.
```